src/rdbms/mongoose_rdbms_cockroachdb.erl

```diff
@@ -0,0 +1,69 @@
+%%==============================================================================
+%% Copyright 2016 Erlang Solutions Ltd.
```

Comment:
I am not sure if we can use the Apache license, because the whole codebase is GPL... Anyway, for me the license part could be skipped (the project as a whole is licensed), or at least we could change the year to 2024 (but then we would need to update it).